python/spacewalk/spacewalk-backend.changes

```diff
@@ -1,3 +1,4 @@
+- Readjusted the code to fix spacewalk-repo-sync --help command
```

Comment:
Great, thanks for the changes!

I would suggest to be a bit more meaningful and concrete here with the message, maybe something like:

```suggestion
- Make "spacewalk-repo-sync --help" command to work if reposync is running.
```